src/int.cr

```diff
@@ -475,8 +475,11 @@ struct Int
     u.unsafe_shl shift
   end
 
+  # Returns the least common multiple of `self` and *other*.
+  #
+  # Raises `OverflowError` in case of overflow.
   def lcm(other : Int)
-    (self * other).abs // gcd(other)
+    (self // gcd(other) * other).abs
```

Comment:
Uuuh. 
Assume: self = 5, other = 15. lcm should be 15 since it is a multiple of both 5 and 15, and no lesser number exists for which that is the case. 
5.gcd(15) == 5. Hence with these changes the above code ends up as:
5 // 5 * 3  => 3.

Reply:
Where is that `3` coming from? `other` is 15.

Reply:
Right. I guess this has been a long week. Never mind.

Reply:
Would've been an intersting week if it ended up repealing operator associativity.

Reply:
Well, `//` is only associative when it is guaranteed to actually divide without remainder (which it should be in this case).